processor/src/main/java/org/mapstruct/ap/internal/conversion/StaticParseToStringConversion.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.internal.conversion;
+
+import java.util.Set;
+
+import org.mapstruct.ap.internal.model.common.ConversionContext;
+import org.mapstruct.ap.internal.model.common.Type;
+import org.mapstruct.ap.internal.util.Collections;
+
+/**
+ * Base class for a conversion between a target type <tt>T</tt> and {@link String},
```

Comment:
Minor, update doc  to remove "base class" now. Something like "Handles conversion between..."

Reply:
Indeed you are right. Will update. Thanks